tests/create-testing-library-rule.test.ts

```diff
@@ -0,0 +1,93 @@
+import { createRuleTester } from './lib/test-utils';
```

Comment:
I added this test file to check both `createTestingLibraryRule` + `detectTestingLibraryUtils` together, covering as much edge cases as possible.